client/blocks/product-selector/index.jsx

```diff
@@ -81,6 +81,17 @@ export class ProductSelector extends Component {
 		} );
 	}
 
+	getDescriptionByProduct( product ) {
```

Comment:
Thanks for creating a method for that. This will make it easier to abstract to a subcomponent later if we decide.